[EDU-5876] Refactor: update Edge Firewall integrations guides (#1412)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/integrations/rate-limiting-with-penalty.mdx

<Fragment slot="panel.console">
1. Select the **Rules Engine** tab.
2. Click the **+ Rule Engine** button.
3. Give a name to the rule.
4. Select a *criteria* to run and catch the domains that you want to run the integration on.
  - Example: if `Host` *matches* `yourdomain.com`.
5. Below, select a *behavior* to the *criteria*. In this case, it'll be **Run Function**. 

>     - Select the adequate function according to the name you gave it in the instantiate step.

Reviewer: ```suggestion
    - Select the adequate function according to the name you gave it during the instantiation step.
```

6. Click the **Save** button.

> On the Console, you must now configure your domain so it is protected by your Edge Firewall.

Reviewer: ```suggestion
On the Console, you must now configure your domain so your edge firewall protects it.
```

7. On the **Products menu**, select **Domains**.
8. Click on the domain you want to protect with your Upstash Rate Limiting function.

> 9. In the **Settings** section, click on the `Edge Firewall` selector and choose the Edge Firewall you've just created.

Reviewer: ```suggestion
9. In the **Settings** section, click the `Edge Firewall` selector and choose the edge firewall you created.
```

Discover how to enhance your web application's performance with this tutorial on **Rate Limit with Penalty on Azion + Upstash DB**. Watch now on Azion's YouTube channel.
	

> 	<iframe width="560" height="315" src="https://www.youtube.com/embed/3qsiKK2GzRw?si=Em23FF2i1BHzTV7r" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

Reviewer: ```suggestion
	<iframe width="560" height="315" src="https://www.youtube.com/embed/3qsiKK2GzRw?si=Em23FF2i1BHzTV7r" loading="lazy" title="How to install Upstash Rate Limiting integration through Azion Marketplace" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
